generated/nidmm/session.py

```diff
@@ -537,9 +537,6 @@ def __setattr__(self, key, value):
             raise AttributeError("'{0}' object has no attribute '{1}'".format(type(self).__name__, key))
         object.__setattr__(self, key, value)
```

Comment:
What about `setattr`?

Reply:
That has a different purpose that has not gone away. We do not want `session.rnge = 5.0` (notice the misspelling) to work.